Makefile

```diff
@@ -26,7 +26,7 @@ else
 # the rest of the Makefile is only visible to Cargo
 
 cargo-prep: $(OUT_DIR)/config.rs
-	@echo "cargo:rustc-flags=-l $(LUA_LIBNAME) -L native=$(LUA_LIBDIRS)"
+	@echo "cargo:rustc-flags=-l $(LUA_LIBNAME) -L $(LUA_LIBDIRS)"
```

Comment:
Normally you don't make a pull request from your `master` branch. You create a different branch for the pull request instead. That way you can continue developing on your `master` without affecting the PR.